versions.json

```diff
@@ -11,6 +11,7 @@
   "rpcs3": { "id": "rpcs3", "code": "RPCS3", "version": 0 },
   "yuzu": { "id": "yuzu", "code": "Yuzu", "version": 1 },
   "ryujinx": { "id": "ryujinx", "code": "Ryujinx", "version": 4 },
+  "shadps4": { "id": "shadps4", "code": "ShadPS4", "version": 0 },
```

Comment:
Unsure if exact semantic version (currently `0.4.0`) will be required here yet...